### PR TITLE
Bugfix/missing binaries

### LINKED ISSUE #27: Binaries are not bundled when building locally
@fcollonval Your expertise is required here.
When I do `pip install .` followed by `pythonfmu-builder -f path/to/slave.py` the produced FMU does not contain binaries for the current platform. I can't run the integration tests either due to this. What gives? Why does it work in the CI? Intended behavior?

### PR DESCRIPTION
**Pull request by markaren**
Closes #27

### CODE REVIEW

**Comment by markaren** [comment]
An issue is that it does not handle 32bit builds..

File: pythonfmu/pythonfmu-export/CMakeLists.txt [hunk 1]
```diff
@@ -67,3 +67,15 @@ target_link_libraries(pythonfmu-export
         PRIVATE
         ${Python3_LIBRARIES}
         )
+
```
Comment (fcollonval) [inline pythonfmu/pythonfmu-export/CMakeLists.txt]:
```suggestion

if(NOT CMAKE_BITNESS)
    set(CMAKE_BITNESS 64)
endif()

```

File: pythonfmu/pythonfmu-export/CMakeLists.txt [hunk 2]
```diff
@@ -67,3 +67,15 @@ target_link_libraries(pythonfmu-export
...
+if (WIN32)
+  set_target_properties(pythonfmu-export
+          PROPERTIES
+          RUNTIME_OUTPUT_DIRECTORY_RELEASE "${CMAKE_CURRENT_SOURCE_DIR}/../resources/binaries/win64"
```
Comment (fcollonval) [inline pythonfmu/pythonfmu-export/CMakeLists.txt]:
```suggestion
          RUNTIME_OUTPUT_DIRECTORY_RELEASE "${CMAKE_CURRENT_SOURCE_DIR}/../resources/binaries/win${CMAKE_BITNESS}"
```

File: pythonfmu/pythonfmu-export/CMakeLists.txt [hunk 3]
```diff
@@ -67,3 +67,15 @@ target_link_libraries(pythonfmu-export
...
+          )
+elseif (UNIX)
+  set_target_properties(pythonfmu-export
+          PROPERTIES
+          LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/../resources/binaries/linux64"
```
Comment (fcollonval) [inline pythonfmu/pythonfmu-export/CMakeLists.txt]:
```suggestion
          LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/../resources/binaries/linux${CMAKE_BITNESS}"
```

File: pythonfmu/pythonfmu-export/CMakeLists.txt [hunk 4]
```diff
@@ -67,3 +67,15 @@ target_link_libraries(pythonfmu-export
...
+          )
+endif ()
```
Comment (fcollonval) [inline pythonfmu/pythonfmu-export/CMakeLists.txt]:
```suggestion
endif ()

```

**Comment by fcollonval** [comment]
Note: in FMPy cmake file, they use:

```cmake
if (WIN32)
   set(FMI_PLATFORM win)
elseif (APPLE)
   set(FMI_PLATFORM darwin)
else ()
   set(FMI_PLATFORM linux)
endif ()

if ("${CMAKE_SIZEOF_VOID_P}" STREQUAL "8")
    set (FMI_PLATFORM ${FMI_PLATFORM}64)
else ()
    set (FMI_PLATFORM ${FMI_PLATFORM}32)
endif ()
```

**Comment by markaren** [comment]
I'm happy with this solution, but I do not know if `LIBRARY_OUTPUT_DIRECTORY` is valid for `darwin`..

**Comment by markaren** [comment]
Phew, quite some improvements today. Lost a watcher though :P

**Pull request merged**